sphinx/themes/basic/static/searchtools.js

```diff
@@ -97,6 +97,14 @@ const _displayItem = (item, searchTerms, highlightTerms) => {
     fetch(requestUrl)
       .then((responseData) => responseData.text())
       .then((data) => {
+
+        // exclude results that don't contain exact phrases if we are searching for them
+        if (data) {
+          const lowercaseData = data.toLowerCase();
+          const mismatch = (s) => !lowercaseData.includes(s);
+          if (exactSearchPhrases.some(mismatch)) return;
```

Comment:
Perhaps `every` would be better than `some` here?

If searching for two phrases, it could be frustrating not to find any results at all, despite the fact that some pages do include one of the phrases.